docs/content/scripting/examples/callback.qml

```diff
@@ -0,0 +1,39 @@
+import QtQml 2.0
+import com.qownnotes.noteapi 1.0
+
+/**
+ * This script is just an example of how to use scripts
+ * Visit http://docs.qownnotes.org/ for more information about scripting
+ */
+QtObject {
+    /**
+     * Just show some log entries when the script is initialized
+     */
+
+    function log(txt){
+     script.log("[callback-example] "+txt);
```

Comment:
Please space before and after `+`

Reply:
I can't see any spaces. :grin:

Reply:
And please use 4 spaces for indenting all the time (not 4 and 2)

Reply:
Wrong file :see_no_evil: